### ISSUE TITLE
Leo's console GUI not longer working in 6.0 beta 1

### ISSUE THREAD

**Issue by ranvik14** [leo-editor/leo-editor#1223]
Contrary to 5.9, where I was able to resolve the initial problem with "leo-curses", it is no longer working in 6.0 beta 1.

According to the traceback messages (see Log-1b) it's related to the recent changes related to moving to Python 3 only ...

<Log-1a> Success in 5.9

(VE4Leo) PS C:\Users\Viktor\pyve\VE4Leo> pip install windows-curses
Collecting windows-curses
  Downloading https://files.pythonhosted.org/packages/1e/75/e390d806df17b30f3617561dc1d6782d68ca608244cdee570019e3e56452/windows_curses-1.1-cp36-cp36m-win_amd64.whl (68kB)
     |████████████████████████████████| 71kB 770kB/s
Installing collected packages: windows-curses
Successfully installed windows-curses-1.1
(VE4Leo) PS C:\Users\Viktor\pyve\VE4Leo> leo-console

cursesGui.py: Tk required for clipboard handling.
loading npyscreen
Leo 5.9-b2, build 20190409061733, Tue Apr  9 06:17:32 UTC 2019
wrote C:/Users/Viktor/.leo/leo.session
(VE4Leo) PS C:\Users\Viktor\pyve\VE4Leo>

</Log-1a>

<Log-1b> Failure in 6.0b1

(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1> pip install windows-curses
Collecting windows-curses
  Using cached https://files.pythonhosted.org/packages/1e/75/e390d806df17b30f3617561dc1d6782d68ca608244cdee570019e3e56452/windows_curses-1.1-cp36-cp36m-win_amd64.whl
Installing collected packages: windows-curses
Successfully installed windows-curses-1.1
(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1> leo-console
cursesGui.py: Tk required for clipboard handling.
Traceback (most recent call last):

  File "c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages\leo\plugins\cursesGui2.py", line 36, in <module>
    from tkinter import Tk # Python 3

ModuleNotFoundError: No module named 'tkinter'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages\leo\core\leoApp.py", line 1018, in createCursesGui
    import leo.plugins.cursesGui2 as cursesGui2

  File "c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages\leo\plugins\cursesGui2.py", line 39, in <module>
    from Tkinter import Tk # Python 2

ModuleNotFoundError: No module named 'Tkinter'

can not create curses gui.
(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1>

</Log-1b>

**Comment by edreamleo** [comment]
Fixed at rev 7b322eb by changing `import _curses` to `import curses` in one place.

**Comment by edreamleo** [comment]
Correction: Fixed at 892d657.  The other fix applies to another issue.

**Comment by ranvik14** [comment]
Hello Edward,

In my environment the problem is still reproducable. - Here's the log of what I did:

< Log >

(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1> pip show leo                                             Name: leo
Version: 6.0b1
Summary: An IDE, PIM and Outliner
Home-page: http://leoeditor.com
Author: Edward K. Ream
Author-email: edreamleo@gmail.com
License: MIT License
Location: c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages
Requires: pyflakes, sphinx, pylint, future, meta, docutils, PyQt5, flexx, nbformat, PyQtWebEngine, six
Required-by:
(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1> pip install https://github.com/leo-editor/leo-editor/archive/devel.zip                                                                                                           Collecting https://github.com/leo-editor/leo-editor/archive/devel.zip
  Downloading https://github.com/leo-editor/leo-editor/archive/devel.zip
     | 96.8MB 1.7MB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: PyQt5; python_version >= "3.0" in c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages (from leo===6.0-b2-devel) (5.12.2)

< Similar content deleted >

Requirement already satisfied: lazy-object-proxy in c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages (from astroid<3,>=2.2.0->pylint->leo===6.0-b2-devel) (1.4.1)
Building wheels for collected packages: leo
  Building wheel for leo (PEP 517) ... done
  Stored in directory: C:\Users\Viktor\AppData\Local\Temp\pip-ephem-wheel-cache-5qoeqhhu\wheels\d2\38\47\90bbdcc2f8e1239dfc04c020e018ad7d03b692145ce0bb74d9
Successfully built leo
Installing collected packages: leo
  Found existing installation: leo 6.0b1
    Uninstalling leo-6.0b1:
      Successfully uninstalled leo-6.0b1
Successfully installed leo-6.0-b2-devel
(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1> pip show leo
Name: leo
Version: 6.0-b2-devel
Summary: An IDE, PIM and Outliner
Home-page: http://leoeditor.com
Author: Edward K. Ream
Author-email: edreamleo@gmail.com
License: MIT License
Location: c:\users\viktor\pyve\ve-for-leo-6-0-b1\lib\site-packages
Requires: docutils, pylint, six, flexx, nbformat, sphinx, pyflakes, meta, PyQtWebEngine, PyQt5, future
Required-by:
(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1> leo-console                                              
Traceback (most recent call last):

  File "c:\users\viktor\appdata\local\programs\python\python36\lib\site-packages\leo\core\leoApp.py", line 1009, in createCursesGui
    import _curses

ModuleNotFoundError: No module named '_curses'

can not import _curses.
Windows: pip install windows-curses
(VE-for-Leo-6-0-b1) PS C:\Users\Viktor\pyve\VE-for-Leo-6-0-b1>  

</ Log >

**Comment by ranvik14** [comment]
@edreamleo - and - @maphew : I just noticed something weird! - Can you explain why the traceback references 

File "c:\users\viktor**\appdata\local\programs\python\python36\**lib\site-packages\leo\core\leoApp.py", line 1009, in createCursesGui

and not

File "c:\users\viktor**\pyve\VE-for-Leo-6-0-b1\**lib\site-packages\leo\core\leoApp.py" ?

**Comment by edreamleo** [comment]
On Sat, Jun 22, 2019 at 2:06 AM ranvik14 <notifications@github.com> wrote:

> @edreamleo <https://github.com/edreamleo> - and - @maphew
> <https://github.com/maphew> : I just noticed something weird! - Can you
> explain why the traceback references
>
> File
> "c:\users\viktor**\appdata\local\programs\python\python36**lib\site-packages\leo\core\leoApp.py",
> line 1009, in createCursesGui
>
> and not
>
> File
> "c:\users\viktor**\pyve\VE-for-Leo-6-0-b1**lib\site-packages\leo\core\leoApp.py"
> ?
>

On the contrary, this is something you should explain to us :-)  There is
something wrong with your (virtual?) environment.

Edward

**Comment by maphew** [comment]
`leo-console` wrapper script is actually missing from current Leo devel. I haven't dug into yet when it was dropped, but I think it was weeks to months ago. On my mental todo but not in GH issues yet.

Try `pip uninstall leo` then re-install.

That aside, `python ...\code\leo-editor\launchLeo-console.py` (which is all `leo-console` does) works for me on `Leo 6.0-b2-devel, devel branch, build cd52639077   2019-07-07 06:26:07 -0700`; Win10, Py3.6.2.

You may need to nuke this VE and start a new one.
